src/goto-programs/Makefile

```diff
@@ -27,6 +27,7 @@ SRC = basic_blocks.cpp \
       interpreter.cpp \
       interpreter_evaluate.cpp \
       json_goto_trace.cpp \
+      lazy_goto_model.cpp \
```

Comment:
This isn't in the right commit, which will make `git bisect` harder.

Reply:
Well caught. This was originally the commit where the cpp file was introduced but I pushed it back to an earlier commit without updating the Makefile. Mea culpa.